### PR TITLE
Extract rgb frames without denseflow

### PR DESCRIPTION
**Pull request by dreamerlin**
1. Add `--use-opencv` args in `build_rawframes.py` to extract rgb rawframes by OpenCV instead of denseflow
2. Rename sth to MMACTION2

### CODE REVIEW

**Comment by innerlee** [comment]
It will be good to support resize in the opencv version of frame extraction.

The benefit of using denseflow is to make sure that the number of flows and the numbers of frames do match exactly.

File: tools/data/build_rawframes.py [hunk 1]
```diff
@@ -29,10 +30,28 @@ def extract_frame(vid_item, dev_id=0):
 
     if task == 'rgb':
         if args.use_opencv:
+            # Not like using denseflow,
+            # Use OpenCV will not make a sub directory with the video name
+            video_name = vid_path.split('.')[0].split('/')[-1]
```
Comment (innerlee) [inline tools/data/build_rawframes.py:35]: use os.path.basename or os.path.splitext to manipulate paths. avoid string operations when possible

**Comment by innerlee** [comment]
note to future: separate different changes into different prs. this pr is a bad example, which mixes name changes and function changes.

**Comment by dreamerlin** [comment]
> note to future: separate different changes into different prs. this pr is a bad example, which mixes name changes and function changes.

You got that right

File: tools/data/build_rawframes.py [hunk 2]
```diff
@@ -19,21 +23,50 @@ def extract_frame(vid_item, dev_id=0):
     """
     full_path, vid_path, vid_id, method, task = vid_item
     if ('/' in vid_path):
-        vid_name = vid_path.split('.')[0].split('/')[0]
-        out_full_path = osp.join(args.out_dir, vid_name)
+        act_name = osp.splitext(vid_path)[0].split('/')[0]
```
Comment (innerlee) [inline tools/data/build_rawframes.py:26]: why there is still `split` present?

File: tools/data/build_rawframes.py [hunk 3]
```diff
@@ -19,21 +23,50 @@ def extract_frame(vid_item, dev_id=0):
     """
     full_path, vid_path, vid_id, method, task = vid_item
     if ('/' in vid_path):
-        vid_name = vid_path.split('.')[0].split('/')[0]
-        out_full_path = osp.join(args.out_dir, vid_name)
+        act_name = osp.split(osp.splitext(vid_path)[0])[0]
```
Comment (innerlee) [inline tools/data/build_rawframes.py:26]: https://www.geeksforgeeks.org/python-os-path-basename-method/

**Pull request merged**